libs/genai/langchain_google_genai/llms.py

```diff
@@ -143,6 +144,10 @@ class _BaseGoogleGenerativeAI(BaseModel):
        not return the full n completions if duplicates are generated."""
     max_retries: int = 6
     """The maximum number of retries to make when generating."""
+
+    timeout: Optional[float] = None
```

Comment:
should we also support `api_transport` parameter?

Reply:
This is in the other PR, I openned one per issue but could have been in this one tbf